codegen/sdk-codegen/build.gradle.kts

```diff
@@ -161,13 +165,16 @@ fun discoverServices(): List<AwsService> {
             val serviceApi = service.getTrait(software.amazon.smithy.aws.traits.ServiceTrait::class.java).orNull()
                 ?: error { "Expected aws.api#service trait attached to model ${file.absolutePath}" }
             val (name, version, _) = file.name.split(".")
+            val serviceClientVersions = Node.parse(rootProject.file("ServiceClientVersions.json").readText(Charset.forName("UTF-8")))
+            val packageVersion = serviceClientVersions.expectObjectNode().getStringMember(name).orNull()?.value ?:
+                serviceClientVersions.expectObjectNode().getStringMember("sdk").get().value
```

Comment:
Reads the package version out of the new `ServiceClientVersions.json` file, then sets it in the smithy-build json so it can be code-generated into the service client.